Formula/leiningen.rb

```diff
@@ -13,7 +13,7 @@ class Leiningen < Formula
   end
 
   resource "jar" do
-    url "https://github.com/technomancy/leiningen/releases/download/2.8.1/leiningen-2.8.1-standalone.zip", :using => :nounzip
```

Comment:
Why is this one being removed?

Reply:
This `.zip` is actually a `.jar`, which is auto-detected.